config/locales/en.yml

```diff
@@ -708,6 +709,44 @@ en:
       <<: *player_form
     show:
       title: Embeddable Player
+  episode_transcripts:
+    form:
+      confirm: Discard unsaved changes?
+    form_status:
+      <<: *form_status
+      title: Transcript Status
+    form_transcript:
+      title: Transcript
+      help:
+        transcript: You can upload your transcript here. Current formats allowed are text and json.
```

Comment:
"Upload a JSON or TXT file containing the text transcript"

(Note to self, once we have a help doc, we should link to it)